src/Compilers/VisualBasic/Portable/Parser/ParseExpression.vb

```diff
@@ -1393,66 +1393,73 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         ' Lines: 16425 - 16425
         ' .Parser::ParseArguments( [ _In_ ParseTree::ArgumentList** Target ] [ _Inout_ bool& ErrorInConstruct ] )
 
-        Private Function ParseArguments(ByRef unexpected As GreenNode, Optional RedimOrNewParent As Boolean = False) As CodeAnalysis.Syntax.InternalSyntax.SeparatedSyntaxList(Of ArgumentSyntax)
+        Private Function ParseArguments(ByRef unexpected As GreenNode, Optional RedimOrNewParent As Boolean = False, Optional attributeListParent As Boolean = False) As CodeAnalysis.Syntax.InternalSyntax.SeparatedSyntaxList(Of ArgumentSyntax)
```

Comment:
>attributeListParent [](start = 126, length = 19)

We might want to add an overload of SyntaxFactory.ParseArgumentList that allows to pass value for this new parameter. We should also make sure attribute binding can handle the new argument list. Both don't have to be addressed in this PR.